src/renderer_mtl.mm

```diff
@@ -3183,7 +3183,7 @@ void writeString(bx::WriterI* _writer, const char* _str)
 		}
 		if (@available(macOS 10.13.2, *))
 		{
-			m_metalLayer.maximumDrawableCount = (_flags&BGFX_RESET_VSYNC) ? 2 : 3;
```

Comment:
Reading docs about `maximumDrawableCount`:
https://developer.apple.com/documentation/quartzcore/cametallayer/2938720-maximumdrawablecount

Seems to match this:
https://github.com/bkaradzic/bgfx/blob/2ec34699af086c00fbea165c4467b9df44496833/include/bgfx/bgfx.h#L636

Reply:
I guess these two settings are related in other backends and Metal just has the one setting?

I changed it to use maxFrameLatency instead. Let me know if that looks good.